magento/pwa-studio
# Hide mini cart on the checkout page.

## Description

Hide the new mini cart on the checkout page.

## Related Issue
Closes [PWA-737](https://jira.corp.magento.com/browse/PWA-737)

### Verification Stakeholders
@soumya-ashok 
@schensley 
@jimbo 

### Verification Steps
1. Go to the checkout page on the deployed app. Cart trigger should not be visible.

## Screenshots / Screen Captures (if appropriate)

![image](https://user-images.githubusercontent.com/35203638/87478776-ef3f9a80-c5ef-11ea-901c-111fc0039e26.png)

![image](https://user-images.githubusercontent.com/35203638/87478829-01b9d400-c5f0-11ea-829c-413662fb9a87.png)

## Checklist
* I have updated the documentation accordingly, if necessary.
